nilearn/reporting/_visual_testing/_glm_reporter_visual_inspection_suite_.py

```diff
@@ -9,220 +9,278 @@
 
-    fmri_glm = FirstLevelModel(mask_img=data['mask'], minimize_memory=True)
+    fmri_glm = FirstLevelModel(mask_img=data["mask"], minimize_memory=True)
     fmri_glm = fmri_glm.fit(fmri_img, design_matrices=design_matrices)
 
     n_columns = design_matrices[0].shape[1]
 
     contrasts = {
-        'SStSSp_minus_DStDSp': _pad_vector([1, 0, 0, -1], n_columns),
-        'DStDSp_minus_SStSSp': _pad_vector([-1, 0, 0, 1], n_columns),
-        'DSt_minus_SSt': _pad_vector([-1, -1, 1, 1], n_columns),
-        'DSp_minus_SSp': _pad_vector([-1, 1, -1, 1], n_columns),
-        'DSt_minus_SSt_for_DSp': _pad_vector([0, -1, 0, 1], n_columns),
-        'DSp_minus_SSp_for_DSt': _pad_vector([0, 0, -1, 1], n_columns),
-        'Deactivation': _pad_vector([-1, -1, -1, -1, 4], n_columns),
-        'Effects_of_interest': np.eye(n_columns)[:5]
+        "SStSSp_minus_DStDSp": _pad_vector([1, 0, 0, -1], n_columns),
+        "DStDSp_minus_SStSSp": _pad_vector([-1, 0, 0, 1], n_columns),
+        "DSt_minus_SSt": _pad_vector([-1, -1, 1, 1], n_columns),
+        "DSp_minus_SSp": _pad_vector([-1, 1, -1, 1], n_columns),
+        "DSt_minus_SSt_for_DSp": _pad_vector([0, -1, 0, 1], n_columns),
+        "DSp_minus_SSp_for_DSt": _pad_vector([0, 0, -1, 1], n_columns),
+        "Deactivation": _pad_vector([-1, -1, -1, -1, 4], n_columns),
+        "Effects_of_interest": np.eye(n_columns)[:5],
     }
-    report = make_glm_report(fmri_glm, contrasts,
-                             bg_img=mean_img_,
-                             height_control='fdr',
-                             )
-    output_filename = 'generated_report_flm_fiac.html'
+    report = make_glm_report(
+        fmri_glm,
+        contrasts,
+        bg_img=mean_img_,
+        height_control="fdr",
+    )
+    output_filename = "generated_report_flm_fiac.html"
     output_filepath = os.path.join(REPORTS_DIR, output_filename)
     report.save_as_html(output_filepath)
     report.get_iframe()
 
 
 ###########################################################################
 
-def _make_design_matrix_slm_oasis(oasis_dataset,
-                                  n_subjects):  # pragma: no cover
-    age = oasis_dataset.ext_vars['age'].astype(float)
-    sex = oasis_dataset.ext_vars['mf'] == b'F'
+
+def _make_design_matrix_slm_oasis(
+    oasis_dataset, n_subjects
+):  # pragma: no cover # noqa
+    age = oasis_dataset.ext_vars["age"].astype(float)
+    sex = oasis_dataset.ext_vars["mf"] == b"F"
     intercept = np.ones(n_subjects)
-    design_matrix = pd.DataFrame(np.vstack((age, sex, intercept)).T,
-                                 columns=['age', 'sex', 'intercept'])
-    design_matrix = pd.DataFrame(design_matrix, columns=['age', 'sex',
-                                                         'intercept']
-                                 )
+    design_matrix = pd.DataFrame(
+        np.vstack((age, sex, intercept)).T, columns=["age", "sex", "intercept"]
+    )
+    design_matrix = pd.DataFrame(
```

Comment:
I'm lost. What does this do ?

Reply:
Would you be fine if I opened an issue to keep track of clarifying this in another PR?
Would prefer to stick to "pure" formatting things in this PR.

Reply:
Yes.